cdc/owner/ddl_manager.go

```diff
@@ -184,6 +198,18 @@ func (m *ddlManager) tick(
 	checkpointTs model.Ts,
 	tableCheckpoint map[model.TableName]model.Ts,
 ) ([]model.TableID, *schedulepb.BarrierWithMinTs, error) {
+	// needBootstrap is true when the the downstream is kafka
+	// and the protocol is simple protocol.
+	if m.needBootstrap {
```

Comment:
it is more appropriate to make it configurable, since other scenarios require this feature as well. Ref: https://github.com/pingcap/tiflow/issues/9419.

Reply:
It is configured by a parameters when the ddl_manager was created.